maglev.tmux

```diff
@@ -11,7 +11,9 @@ fi
 SHOW_BATTERY=false
 if [[ $PLUGINS == *"tmux-battery"* ]]; then
     # if no battery can be found but the plugin is still enabled, this might result in some blank space in status-right
-    SHOW_BATTERY=true
+    if [ ! x"$(uname -s)" = x"Linux" ] || [[ "$(ls /sys/class/power_supply)" == *"BAT"* ]]; then
```

Comment:
Turns out, this is how it works later on in the battery function.
While I believe this isn't perfect, it should work on linux as long as battery drivers work properly.

No Idea what happens on macOS without battery (iMac, Mac Pro) or how to handle it if necessary.